go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -268,6 +266,27 @@ func (c *Controller) syncServices(key string) error {
 			"Error listing Endpoint Slices for Service %s/%s: %v", namespace, name, err)
 		return err
 	}
+
+	// If idling enabled and there are no endpoints, we need to move the VIP from the main loadbalancer
+	// ,that has the reject option for backends without endpoints, to the idling loadbalancer, that
+	// generates an event to kick of the unidling process in the openshift controller manager.
+	// if not, we delete the vips from the idling lb and move them to their right place.
```

Comment:
that is not covering all the cases, there can be endpointslices , but no endpoints ready, so it is possible, not common, that `getLbEndpoints()` later return you 0 endpoints for an specific vip or all vips

Reply:
Changed the logic trying to see if there's at least one valid ip